ACMAS/.env.dev

```diff
@@ -0,0 +1,10 @@
+DATABASE=postgres
+DEBUG=1
+DJANGO_ALLOWED_HOSTS='localhost 127.0.0.1 [::1]'
+SECRET_KEY=foo
```

Comment:
Is this something user specific and should be hidden from here?

Reply:
This is a file with environment variables specifically for development purposes, and thus doesn't need to be hidden.